### PR TITLE
Eip712 verify

### PR DESCRIPTION
**Pull request by markrypt0**

### CODE REVIEW

**Review by BitHighlander — APPROVED** [review]
:shipit:

File: eip712msg.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{"types":{"EIP712Domain":[{"name":"name","type":"string"},{"name":"version","type":"string"},{"name":"chainId","type":"uint256"},{"name":"verifyingContract","type":"address"}],"Permit":[{"name":"owner","type":"address"},{"name":"spender","type":"address"},{"name":"value","type":"uint256"},{"name":"nonce","type":"uint256"},{"name":"deadline","type":"uint256"}]},"domain":{"name":"USD Coin","version":"2","verifyingContract":"0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48","chainId":1},"primaryType":"Permit","message":{"owner":"0x33b35c665496bA8E71B22373843376740401F106","spender":"0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45","value":"4023865","nonce":0,"deadline":1655431026},
```
Comment (pastaghost) [inline eip712msg.json:1]: Replace address at "owner" field with address generated from "all all all" test mnemonic

File: eip712msg.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "domain": {
+        "name": "USD Coin",
+        "version": "2",
+        "verifyingContract": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        "chainId": 1
+    },
+    "primaryType": "Permit",
+    "message": {
+        "owner": "0x33b35c665496bA8E71B22373843376740401F106",
```
Comment (pastaghost) [inline eip712msg.json:25]: Replace "owner" address with address derived from "all all all" test mnemonic

**Review by pastaghost — APPROVED** [review]
Maybe consider moving eip712msg.json into new test data subdirectory, otherwise lgtm.

**Pull request merged**